src/widgets/block.rs

```diff
@@ -221,38 +335,81 @@ impl<'a> Block<'a> {
         self.title_position(Position::Bottom)
     }
 
-    /// Positions all titles that don't have a position
+    /// Positions all [titles](Title) that don't have a [`Position`].
+    ///
+    /// This effectively acts as a default for titles that don't explicitly set an [`Position`].
+    ///
     /// # Example
+    ///
     /// This example position all titles on the bottom except "top" title
     /// ```
     /// # use ratatui::widgets::{Block, BorderType, Borders, block::title::{Position, Title}};
     /// Block::default()
-    ///   // This title won't be aligned in the center
-    ///   .title(Title::from("top").position(Position::Top))
-    ///   .title("foo")
-    ///   .title("bar")
-    ///   .title_position(Position::Bottom);
+    ///     // This title won't be aligned in the center
+    ///     .title(Title::from("top").position(Position::Top))
+    ///     .title("foo")
+    ///     .title("bar")
+    ///     .title_position(Position::Bottom);
     /// ```
     pub const fn title_position(mut self, position: Position) -> Block<'a> {
         self.titles_position = position;
         self
     }
 
+    /// Defines the style of the borders.
+    ///
+    /// If a [`Block::style`] is defined, `border_style` will be applied on top of it.
```

Comment:
The order of the styles being applied might be worth putting in the widget level doc rather than just the method level